end_to_end/tpu/gemma/9b/2_test_gemma.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# This file, combined with step 2 in the same directory, demonstrates converting a Gemma checkpoint from Kaggle and running various MaxText operations on it.
+# This step is tested nightly on an ordinary CPU VM.
```

Comment:
Can you change this comment and pick it up from https://github.com/google/maxtext/blob/main/end_to_end/tpu/gemma/7b/2_test_gemma.sh

Reply:
Done